examples/fluids/meshes/cylinder.geo

```diff
@@ -35,23 +38,23 @@ DefineConstant[
 ];
 
 // Exterior corners
-Point(1) = {-xL, -H, 0, 1.0};
-Point(2) = {xL, -H, 0, 1.0};
-Point(3) = {xR, -H, 0, 1.0};
-Point(4) = {-xL, H, 0, 1.0};
-Point(5) = {xL, H, 0, 1.0};
-Point(6) = {xR, H, 0, 1.0};
+Point(1) = {-xL, -H, 0};
+Point(2) = {xL, -H, 0};
+Point(3) = {xR, -H, 0};
+Point(4) = {-xL, H, 0};
+Point(5) = {xL, H, 0};
+Point(6) = {xR, H, 0};
 
 // Coordinates for Cylinder points
-Point(7) = {-cyldiameter/4, -cyldiameter/4, 0, 1.0};
+Point(7) = {-cyldiameter/4, -cyldiameter/4, 0};
 //+
-Point(8) = {cyldiameter/4, -cyldiameter/4, 0, 1.0};
+Point(8) = {cyldiameter/4, -cyldiameter/4, 0};
 //+
-Point(9) = {-cyldiameter/4, cyldiameter/4, 0, 1.0};
+Point(9) = {-cyldiameter/4, cyldiameter/4, 0};
 //+
-Point(10) = {cyldiameter/4, cyldiameter/4, 0, 1.0};
+Point(10) = {cyldiameter/4, cyldiameter/4, 0};
```

Comment:
Basic trigonometry question: If you have a circle of radius $r = d/2$ centered at the origin, what is the coordinate of the point at a 45 degree angle ($\pi/4$ radians) on the unit circle?

Reply:
The coordinate at 45 degree on a unit circle will be $(\sqrt{2}/2, \sqrt{2}/2)$